libpod/container_internal.go

```diff
@@ -1404,11 +1404,8 @@ func (c *Container) cleanupStorage() error {
 	var cleanupErr error
 
 	for _, containerMount := range c.config.Mounts {
-		if err := c.unmountSHM(containerMount); err != nil {
-			if cleanupErr != nil {
-				logrus.Errorf("Error unmounting container %s: %v", c.ID(), cleanupErr)
-			}
-			cleanupErr = err
+		if err := c.unmountExtra(containerMount); err != nil {
+			return err
```

Comment:
Same here. Can we preserve the wrapping?

Reply:
We also can't just return here. We need to logrus this and set cleanupErr.

Reply:
Wrapping happens lower, and I am no longer returning the error, just logging it.